docs/reference/glossary.md

```diff
@@ -176,17 +202,29 @@ This is how long a client waits for a response from the broker after the client
 
 - [Zeebe API (gRPC)](/apis-tools/grpc.md)
 
+### Segment
+
+The log consists of one or more segments. Each segment is a file containing an ordered sequence records. Segments are deleted when the log is compacted.
+
+- [Resource planning](/self-managed/zeebe-deployment/operations/resource-planning.md#event-log)
+
 ### Snapshot
 
 The state of all active process instances, (these are also known as inflight process instances) are stored as records in an in-memory database called RocksDB. A snapshot represents a copy of all data within the in-memory database at any given point in time. Snapshots are binary images stored on disk and can be used to restore execution state of a process. The size of a snapshot is affected by the size of the data. Size of the data depends on several factors, including complexity of the model or business process, the size and quantity of variables in each process instance, and the total number of executing process instances in a broker.
 
 - [Resource planning](/self-managed/zeebe-deployment/operations/resource-planning.md#snapshots)
 
-### Segment
+### Subscription
 
-The log consists of one or more segments. Each segment is a file containing an ordered sequence records. Segments are deleted when the log is compacted.
+An [inbound connector](#inbound-connector) that subscribes to a message queue.
 
-- [Resource planning](/self-managed/zeebe-deployment/operations/resource-planning.md#event-log)
+This way, a Camunda workflow can receive messages from an external system or service (like Kafka, or RabbitMQ) using message queuing technology. This type of inbound connector is commonly used in distributed systems where different components of the system need to communicate with each other asynchronously.
+
+### Webhook
```

Comment:
same here, i would add connector here, maybe even inbound connector

Reply:
Interested in your thoughts here @akeller, as this is a subtype.